examples/03_connectivity/plot_atlas_comparison.py

```diff
@@ -0,0 +1,73 @@
+"""
+Extracting resting-state signals from different atlases for comparison
+======================================================================
+
+In this example :class:`nilearn.input_data.NiftiLabelsMasker` is used to
+extract time series from nifti objects using different parcellation atlases.
```

Comment:
An important part of the example is to turn a parcellation into a connectome for visualization. This requires choosing centers of each parcel via  plotting.find_parcellation_cut_coords.

Reply:
We should demo also find_probabilistic_atlas_cut_coords in the same example, in a second section. Hence, we should rephrase the above as "An important part of the example is to turn an atlas into a connectome for visualization. The requires choosing centers for each parcel or network, via plotting.find_parcellation_cut_coords or plotting.find_probabilistic_atlas_cut_coords depending on whether the atlas is a parcellation or a set of maps.